### DISCUSSION TITLE
Fastcgi error with DevTools

### DISCUSSION THREAD

**Discussion by marctaule** [Laravel-Backpack/community-forum#970] [Q&A (Help)]
I'm using DevTools and sometimes I get the following error on my apache error log:
[fastcgi:error] [pid 11943] [client ::1:62012] FastCGI: incomplete headers (0 bytes) received from server "/Applications/MAMP/fcgi-bin/php8.2.13.fcgi", referer: http://project.local/admin/dashboard

Anybody knows why do I get this error?

Thanks a lot! 

This is what I get when I make: php artisan backpack:version:

### PHP VERSION:
8.2.4

### PHP EXTENSIONS:
Core, date, libxml, openssl, pcre, sqlite3, zlib, bcmath, bz2, calendar, ctype, curl, dom, hash, fileinfo, filter, ftp, gd, gettext, json, iconv, intl, SPL, ldap, mbstring, session, standard, pcntl, exif, mysqlnd, PDO, pdo_mysql, pdo_sqlite, Phar, posix, random, readline, Reflection, mysqli, SimpleXML, soap, sockets, sodium, tokenizer, xml, xmlreader, xmlwriter, xsl, zip, imap, pgsql, pdo_pgsql, xdebug

### LARAVEL VERSION:
11.5.0.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.3.3
backpack/crud: 6.7.11
backpack/devtools: 3.1.3
backpack/filemanager: 3.0.7
backpack/generators: v4.0.5
backpack/pro: 2.1.13
backpack/theme-tabler: 1.2.10

**Comment by karandatwani92** [comment]
Hey @marctaule 

Thanks for reporting. It seems related to timeout or x-debug. 

I was not able to reproduce the error. It would be helpful if you could share a way to reproduce it.
And, I've asked my colleague @pxpm to re-check.

**Reply by marctaule** [reply]
Hi @karandatwani92! Thanks for your anwser.
It's very strange because it only happens to me sometimes. 
Maybe it has something to do with my computer... 
Today I turned on the computer and clicked on Devtools, and it worked correctly. 
This happened to me last time too. It was working, then suddenly stopped working, then started working again, and then stopped working again... I wouldn't know how to reproduce it 🥺

**Comment by jcastroa87** [comment]
Due to not activity i will close the issue, but please feel free to re-open or create a new one if needed.

Cheers
